python/paddle/fluid/tests/unittests/test_imperative_numpy_bridge.py

```diff
@@ -39,9 +40,18 @@ def test_tensor_from_numpy(self):
             self.assertTrue(np.array_equal(var2.numpy(), data_np))
             data_np[0][0] = -1
             self.assertEqual(data_np[0][0], -1)
-            self.assertNotEqual(var2[0][0].numpy()[0], -1)
+            if _in_eager_mode():
+                # eager_mode, var2 is EagerTensor, is not subscriptable
```

Comment:
Mark this a todo here to support slice in eager mode later

Reply:
Ok, TODO item will mark in the next PR soon.